src/include/catalog/catalog_content.h

```diff
@@ -74,20 +77,25 @@ class CatalogContent {
 
     std::vector<common::table_id_t> getTableIDs(CatalogEntryType catalogType) const;
 
-    common::table_id_t createNodeTable(const binder::BoundCreateTableInfo& info);
-    common::table_id_t createRelTable(const binder::BoundCreateTableInfo& info);
-    common::table_id_t createRelGroup(const binder::BoundCreateTableInfo& info);
-    common::table_id_t createRDFGraph(const binder::BoundCreateTableInfo& info);
+    common::table_id_t createTable(const binder::BoundCreateTableInfo& info);
     void dropTable(common::table_id_t tableID);
+    void alterTable(const binder::BoundAlterInfo& info);
+
+    // ----------------------------- Internal functions -----------------------
+    std::unique_ptr<CatalogEntry> createNodeTableEntry(
```

Comment:
try make private if these suppose to be internal functions.

Reply:
they are indeed private. just want to separate them from other functions used in `Catalog`. the comment is confusing though. I removed it separate them with another `private` keyword.